core/src/main/java/io/aiven/klaw/controller/AclController.java

```diff
@@ -109,7 +110,7 @@ public ResponseEntity<TopicOverview> getAcls(
       value = "/getSchemaOfTopic",
       method = RequestMethod.GET,
       produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<TopicOverview> getSchemaOfTopic(
+  public ResponseEntity<SchemaOverview> getSchemaOfTopic(
```

Comment:
How about moving this to SchemaRegstryController ?

Reply:
Also how about renaming the TopicOverviewService -> ResourceOverviewService as it deals with Topcis and Schemas?